benchs/bench_all_ivf/cmp_with_scann.py

```diff
@@ -270,6 +270,8 @@ def faiss_eval_search(
           "by residual=", index.by_residual)
 
     print("adding a refine index")
+    flat_index = faiss.IndexFlat(index.d, index.metric_type)
+    flat_index.add(xb)
```

Comment:
where did you use the variable flat_index ?

Reply:
ah right this should be removed